rollup.config.js

```diff
@@ -1,41 +0,0 @@
-import pkg from './package.json';
-import babel from '@rollup/plugin-babel';
-import resolve from '@rollup/plugin-node-resolve';
-import commonjs from '@rollup/plugin-commonjs';
-import clear from 'rollup-plugin-clear';
-import visualizer from 'rollup-plugin-visualizer';
-
-const extensions = [ '.ts', '.tsx' ];
-
-module.exports = {
-    input: './src/index.ts',
-    output: [
-        {
-            file: 'dist/index.js',
-            format: 'cjs',
-        },
-        {
-            file: 'dist/index.esm.js',
-            format: 'esm',
-        },
-    ],
-    external: [
-        // Ignore all dependencies and PeerDependencies in dist
-        ...Object.keys(pkg.dependencies || {}),
```

Comment:
i think if you just want to fix the bundling you can get rollup to exclude emotion's jsx stuff from the build by explicitly adding `@emotion/react/jsx-runtime` to rollup's externals.

this line will ensure `@emotion/react` isn't bundled but it won't match on submodes/paths

Reply:
Thanks @sndrs for your comments. Apart from getting rollup to exclude emotion's jsx, there's the benefit of getting rid of extra dependencies (rollup + babel), inspired by source that seems to do the job! Would it not be best to simplify our *-rendering projects by only relying on the TS compiler if there's no need for an actual bundler and let the the host app bundlers take care of themselves? Not to mention there's a lot less config code one needs to understand/maintain.

Reply:
I guess what I don't know is, what is the benefit of adding a bundler here. Maybe there is a reason that I'm not aware of?

Reply:
yeah i completely agree, i think TS compiler is prob the best option for internal packages, i only wanted to mention it in case you'd have stuck with rollup if you could